dist/form.js

```diff
@@ -1,13 +1,13 @@
 const form = document.querySelector('#form');
 const feedBack = document.querySelector('.feedback');
 
-
-
 form.addEventListener('submit', e => {
   e.preventDefault();
   
   // validation
-  const fullName = form.fullname.value;
+  const fullName = form.fullname.value
+  const email = form.email.value;
+  const text = form.message.value
   const fullNamePattern = /^[A-Za-z]{6,20}$/;  
```

Comment:
- [ ] Like you will hear me speak in the video, I think it is nicer to store all your form input values into a single object so that storing and accessing it in the local storage will be very easy. Something like the example below:
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```